OpenRA.Mods.Common/WorldExtensions.cs

```diff
@@ -45,9 +46,9 @@ public static IEnumerable<Actor> FindActorsOnLine(this World world, WPos lineSta
 			foreach (var currActor in actorsInSquare)
 			{
 				var actorWidth = 0;
-				var healthInfo = currActor.Info.TraitInfoOrDefault<HealthInfo>();
-				if (healthInfo != null)
-					actorWidth = healthInfo.Shape.OuterRadius.Length;
+				var shapeInfos = currActor.Info.TraitInfos<HitShapeInfo>().Where(Exts.IsTraitEnabled);
```

Comment:
`Where(Exts.IsTraitEnabled)` doesn't make sense on TraitInfos.  You will need to enumerate `TraitsImplementing<HitShape>().Where(Exts.IsTraitEnabled)` and then use `.Info.Shape`